man/preupg.1

```diff
@@ -38,9 +48,9 @@ Clean all files created by preupg about previous state of the machine.
 Suppress user interaction.
 .TP
 .B \-\-kickstart
-Generate kickstart.
+Generates kickstart.
```

Comment:
OK... although the grammatical mood is not really consistent: you changed imperative to indicative here, and added imperative few paragraphs later. It would be nice to have it consistent. (Imperative seems to be more common.)